camera_matrix:
- - 499.5331883973887
  - 0.0
  - 629.6263920813349
- - 0.0
  - 499.1701210932231
  - 343.0816351964764
- - 0.0
  - 0.0
  - 1.0
dist_coeff:
- - -0.3706082208349003
  - 0.32552105719380126
  - -0.00027202042466837597
  - 0.0013572055423475566
  - -0.3634729441344056
